cmd/describe/volume.go

```diff
@@ -222,6 +272,24 @@ $ storageos describe volume --namespace my-namespace-name my-volume-name
 			}
 			c.namespace = ns
 
+			if masterHealthState != "" && health.MasterHealthFromString(masterHealthState) == health.Invalid {
+				return fmt.Errorf("master health state not recognized")
+			}
+			if replicaHealthState != "" && health.ReplicaHealthFromString(replicaHealthState) == health.Invalid {
+				return fmt.Errorf("replica health state not recognized")
+			}
+
+			// can still store "invalid" in wanted state but it now means there
+			// was no user input
+			c.masterHealthState = health.MasterHealthFromString(masterHealthState)
+			c.replicasHealthState = health.ReplicaHealthFromString(replicaHealthState)
+
+			// -1 is the default value to indicate no user input
```

Comment:
It might be easier to follow just have an explicit `replicaCountFilterEnabled` variable, instead of encoding it as `-1` in the `replicaCount`